packages/grafana-ui/src/components/Table/FilterList.tsx

```diff
@@ -19,16 +19,16 @@ export const FilterList: FC<Props> = ({ options, values, onChange }) => {
   const theme = useTheme2();
   const styles = getStyles(theme);
   const [searchFilter, setSearchFilter] = useState('');
-  const items = useMemo(() => options.filter((option) => option.label?.indexOf(searchFilter) !== -1), [
-    options,
-    searchFilter,
-  ]);
+  const items = useMemo(
+    () => options.filter((option) => option.label?.toLowerCase().indexOf(searchFilter.toLowerCase()) !== -1),
```

Comment:
nit: maybe change to `includes` instead
```suggestion
    () => options.filter((option) => option.label?.toLowerCase().includes(searchFilter.toLowerCase())),
```